wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -89,6 +90,37 @@ public static function getSubscribedEvents() {
     ];
   }
 
+  /**
+   *
+   */
+  public static function assignVolunteerAsCampInitiator(string $op, string $objectName, $objectId, &$objectRef) {
+    if ($op !== 'edit' || $objectName !== 'AfformSubmission') {
```

Comment:
Can we add a more specific check here for the Collection Camp form? Currently, this runs for all Afforms and causes issues. I expect the hook is being triggered on other forms as well, which results in errors. Can you please check and add an Afform check so that it only applies to the Collection Camp form?